master_plan/Functional_testing/Authentication/tcid-aula01-administrator can login/README.md

```diff
@@ -0,0 +1,14 @@
+## Prerequisites:
```

Comment:
Name this as tcid-au1a01.md

Reply:
Structure could be of the directory
Testpaln -- Masterplan.md
 Testplan/subfeatures1/subfeature1.md <Details of feature and approach>
 Testplan/subfeature1/tcid1.md <Details of actual TC>
 Testplan/subfeature1/tcid2.md <Details of actual TC>
 Testplan/subfeatures2/subfeature2.md <Details of feature and approach>
 Testplan/subfeature2/tcid1.md <Details of actual TC>
 Testplan/subfeature2/tcid2.md <Details of actual TC> 
All TCID should be unique